ethers-providers/src/pending_transaction.rs

```diff
@@ -78,6 +104,22 @@ impl<'a, P: JsonRpcClient> PendingTransaction<'a, P> {
     }
 }
 
+impl<'a, P> PendingTransaction<'a, P> {
+    pub fn inspect<F>(self, mut f: F) -> Self
```

Comment:
```suggestion
    /// Allows inspecting the content of a pending transaction in a builder-like way to avoid
    /// more verbose calls, e.g.: 
    /// `let mined = token.transfer(recipient, amt).send().await?.inspect(|tx| println!(".{}", *tx)).await?;`
    pub fn inspect<F>(self, mut f: F) -> Self
```